xbmc/pvr/addons/PVRClientMenuHooks.cpp

```diff
@@ -26,7 +26,7 @@ CPVRClientMenuHook::CPVRClientMenuHook(const std::string& addonId, const PVR_MEN
       hook.category != PVR_MENUHOOK_TIMER &&
       hook.category != PVR_MENUHOOK_EPG &&
       hook.category != PVR_MENUHOOK_RECORDING &&
-      hook.category != PVR_MENUHOOK_RECORDING &&
+      hook.category != PVR_MENUHOOK_DELETED_RECORDING &&
```

Comment:
This change is unrelated. Please remove this change from this PR and open a separate PR for it.

Reply:
Please rebase your branch to latest upstream master and you will see that this fix already went in.

Reply:
Your branch is based on master from Nov 2020. ;-)

Reply:
`git fetch upstream`
`git rebase upstream/master`
`git push origin -ff <your-branch-name>`

Reply:
yes, overseen that. sorry :)

Reply:
@ksooo I tried but i created two commit when i tried to remove the change in this file.